### ISSUE TITLE
Version mismatch in announcement mail list [enhancement]

### ISSUE THREAD

**Issue by Anon1234** [i3/i3#1537]
```
Hi,

I just released i3 v4.8. Release notes follow:


 ┌──────────────────────────────┐
 │ Release notes for i3 v4.9.1  │
 └──────────────────────────────┘
```

**Comment by stapelberg** [comment]
Yeah, my mistake. Can’t do anything about it, so I’m going to close this.

**Comment by Airblader** [comment]
@stapelberg You did it again. :)

> Hi,
> 
>I just released i3 v4.8. Release notes follow:
>
>
 >┌──────────────────────────────┐
 >│ Release notes for i3 v4.10   │
 >└──────────────────────────────┘

**Comment by stapelberg** [comment]
I’ll add the email sending part to the release.sh script to avoid this in the future.
